test/minikube/minikube_base_database_test.go

```diff
@@ -113,10 +113,10 @@ func verifyPacketFetch(t *testing.T, namespaceName string, admin0 string) {
 	t.Log("tar contents: ", output)
 }
 
-func startDatabase(t *testing.T, namespaceName string, adminPod string, options *helm.Options) {
+func startDatabase(t *testing.T, namespaceName string, adminPod string, options *helm.Options) (helmChartReleaseName string) {
```

Comment:
Ah!
I had considered doing this when I refactored the tests for Backup - great to see it being done!